### PR TITLE
Fix/working test

### PR DESCRIPTION
**Pull request by tintinthong**
- [x]  pointing test to test network
- [x] succesful run of test
- [ ] upgrade truffle, solidity compiler and open zeppelin
- [ ] fixing travis build

### CODE REVIEW

File: truffle.js [hunk 1]
```diff
@@ -4,4 +4,11 @@ require('babel-polyfill');
 module.exports = {
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
+  networks: {
+    test: {
```
Comment (habdelra) [inline truffle.js:8]: I don't think this is necessary-- doesn't `truffle test` already default to this?

**Comment by habdelra** [comment]
Also we're migrating off of travis CI. feel free to add github actions integration for CI testing.

**Comment by habdelra** [comment]
I think  you'll actually want to update to the latest travis first, that will probably solve a bunch of your CI issues. Also, consider using a current version of node, like 14 LTS

File: truffle.js (hunk 1 above)
Comment (tintinthong) [reply truffle.js:8]: true. it is supposed to default to the test-rpc baked into truffle. But, when I ran it, it couldn't find a test network to connect to. I think ill upgrade the truffle and solidity versions.

**Comment by tintinthong** [comment]
> I think you'll actually want to update to a modern version of truffle first, that will probably solve a bunch of your CI issues. like 0.5.17 or later. Also, consider using a current version of node, like 14 LTS. And use volta to pin it

yup yup sounds like a good idea

**Pull request closed without merging**